packages/file-adapters/README.md

```diff
@@ -153,14 +153,14 @@ const fileAdapter = new S3Adapter({
 
 ### Config
 
-| Option         | Type                   | Default     | Description                                                                                                                                                                                                                              |
-| -------------- | ---------------------- | ----------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `bucket`       | `String`               | Required    | S3 bucket name                                                                                                                                                                                                                           |
-| `folder`       | `String`               | `''`        | Upload folder from root of bucket. By default uploads will be sent to the bucket's root folder.                                                                                                                                          |
-| `getFilename`  | `Function`             | `null`      | Function taking a `{ id, originalFilename }` parameter. Should return a string with the name for the uploaded file on disk.                                                                                                              |
-| `publicUrl`    | `Function`             |             | By default the publicUrl returns a url for the S3 bucket in the form `https://{bucket}.s3.amazonaws.com/{key}/{filename}`. This will only work if the bucket is configured to allow public access.                                       |
-| `s3Options`    | `Object`               | `undefined` | For available options refer to the [AWS S3 API](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html)                                                                                                                         |
-| `uploadParams` | `Object    | Function` | `{}`        | A config object or function returning a config object to be passed with each call to S3.upload. For available options refer to the [AWS S3 upload API](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#upload-property). |
+| Option         | Type       | Default     | Description                                                                                                                                                                                        |
```

Comment:
This table seems to be broken(at least on GH)

Reply:
yeah, I'm reverting the change, seems like prettier is still doing funky things with md